src/components/sub-menu/SubMenu.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-menu
+    :close-on-content-click="false"
+    :offset-x="isOffsetX"
+    :offset-y="isOffsetY"
+    :open-on-hover="isOpenOnHover"
+    :transition="transition"
+    :value="openMenu"
+  >
+    <template #activator="{ on }">
+      <v-btn
+        v-if="icon"
+        :color="color"
+        v-on="on"
+      >
+        <v-icon>{{ icon }}</v-icon>
+      </v-btn>
+      <v-list-item
+        v-else-if="isSubMenu"
+        class="d-flex justify-space-between"
+        v-on="on"
+      >
+        {{ name }}
+        <div class="flex-grow-1" />
+        <v-icon>mdi-chevron-right</v-icon>
+      </v-list-item>
+      <v-btn
+        v-else
+        :color="color"
+        text
+        v-on="on"
+        @click="openMenu=true"
+      >
+        {{ name }}
+      </v-btn>
+    </template>
+    <v-list>
+      <template v-for="(item, index) in menuItems">
+        <v-divider
+          v-if="item.isDivider"
+          :key="index"
+        />
+        <sub-menu
```

Comment:
Note the "recursion" here. This results in a submenu to the side of the current menu. What I actually wanted is a "nested" submenu, which opens in the same menu as the parent, like this:

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/09381390-c6b6-4ebd-9643-a1c95605e6ad)

Reply:
I have now created a generic component that does the above ^^ and which I will reuse for Home Jurisdictions. You'll see it in a future PR :)